pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dhl-sdk"
-version = "0.1.0"
+version = "0.2.0"
 description = ""
 authors = ["t.mateus.datahow <t.mateus@datahow.ch>"]
 readme = "README.md"
```

Comment:
should we put a more official email here? check with fabi/adam what a more official email is to relay feedback etc. otherwise ppl might write you directly hihi

additionally poetry can make use of dynamic versioning etc. etc. hooked up via the CI/CD so we dont have to manually update `version`, but obv. that depends on what we can do on github